index.html

```diff
@@ -0,0 +1,168 @@
+    <title>E-com whitepaper</title>
+    <link rel="stylesheet" href="../Sass/style.css" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="../Sass/style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="mobileNav d-md-none">
+        <i class="fa fa-bars pt-4 fa-3x" id="hamburger" aria-hidden="true"></i>
+      </nav>
+      <nav class="desktopNav mt-3">
+        <div
+          class="destopNavItems d-flex align-items-center justify-content-between"
+        >
+          <div class="logo ms-5 d-none d-md-block">
+            <a class="logo" href="./index.html"
+              ><img src="../img/logo.jpeg" alt=""
+            /></a>
+            <br />
+          </div>
+          <div id="desktopList" class="d-none d-md-flex">
+            <img
+              id="closeX"
+              class="d-none ps-5 mb-5"
+              src="../img/close.png"
+              alt=""
+            />
+            <ul
+              class="items list-unstyled d-flex gap-4 align-items-center d-md-flex flex-column flex-md-row pt-5 pt-md-0"
+            >
+              <li class="ulLi ms-5 me-5 p-2">
+                <a
+                  class="text-decoration-none text-danger"
+                  href="../About/about.html"
+                  >About</a
+                >
+              </li>
+              <li class="ulLi">
+                <a class="text-decoration-none text-dark" href="#program"
+                  >Program</a
+                >
+              </li>
+              <li class="ulLi">
+                <a class="text-decoration-none text-dark" href="#speakers"
+                  >Speakers</a
+                >
+              </li>
+              <li class="ulLi">
+                <a
+                  class="text-decoration-none text-dark text-md-dark"
+                  href="#sponsors"
+                  >Sponsors</a
+                >
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+      <h3>"Sharing with the world"</h3>
+      <h1 class="col-md-7">THE PERFECT BLUEPRINT TO E-COMMERCE 2022</h1>
+    </header>
+
+    <main class="col-md-12">
+      <div class="hero">
+        <p class="moment col-md-5 mx-md-5">
+          The long awaited joyful moment beliving in the value of openess and
+          sharing, creating a positive change with different people from accros
+          the world is taking place in August, Abuja Nigeria
+        </p>
+        <div class="mx-md-5">
+          <h2 class="date">2022.08.11(THU) ~ 12(FRI)</h2>
+          <p class="ejay">@ Ojayent e-comm hub</p>
```

Comment:
- [ ] The other reviewer asked that you work on the alignment of the top section on your Home Page. Kindly give some spacing and align the section to the right. You can use the image below tagged **"Design Template"** to make the changes. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/181358877-0278589a-534b-441d-9b64-32d6130f756f.png)

### Design Template

![image](https://user-images.githubusercontent.com/77106962/181358975-799fb2ea-6869-4de0-90bb-7e14c1715996.png)

Reply:
section readjusted towards center